src/markdown-pages/install-with-kurl/migrating-csi.md

```diff
@@ -180,6 +180,43 @@ To migrate to a new CSI provisioner in a kURL cluster:
 
    During the cluster upgrade, the kURL installer detects that the CSI add-on has changed. kURL automatically begins the process of migrating data from the current CSI provisioner to the provisioner in the updated specification. For more information about the data migration process, see [About Changing the CSI Add-on](#about-changing-the-csi-add-on) above.
 
+## Automated Local to Distributed Storage Migrations
+
+kURL is able to migrate clusters from local (Non-HA) storage to distributed (HA) storage when it reaches n number of nodes. After a node greater than or equal to n join the cluster, the join script will prompt to perform the migration. This migration is also available as a task within task.sh.  Please note the following requirements when choosing to have customers on both local and distributed storage.
```

Comment:
>kURL is able to migrate clusters from local (Non-HA) storage to distributed (HA) storage when it reaches n number of nodes

We talked about N=3 being the minimum value and so we should explicitly state this.